templates/client-daemonset.yaml

```diff
@@ -158,13 +166,20 @@ spec:
                 -hcl='leave_on_terminate = true' \
                 {{- if .Values.global.tls.enabled }}
                 -hcl='ca_file = "/consul/tls/ca/tls.crt"' \
+                {{- if .Values.global.tls.enableAutoEncrypt }}
+                -hcl='auto_encrypt = {tls = true}' \
```

Comment:
Good find! Yeah, we can move it to the config map in this case. Do you feel it should be done in this PR or maybe do a sweeping change in a separate PR? If separate, then it might make sense to create an issue for it.

Reply:
Definitely not this pr! Was just thinking about it.